README.rst

```diff
@@ -1,42 +1,59 @@
-=======================
-EOX core |build-status|
-=======================
-
-.. |build-status| image:: https://circleci.com/gh/eduNEXT/eox-core.svg?style=svg
-
-Eox-core (A.K.A. Edunext Open extensions) is an `openedx plugin`_, for the `edx-platform`_ that adds multiple API
-endpoints in order to extend the functionality of the `edx-platform`_ and avoid changing the base code directly. These
-API endpoints includes bulk creation of pre-activated users (for example, skip sending an activation email), enrollments
-and pre-enrollment operations.
+========
+EOX Core
+========
+|Maintainance Badge| |Test Badge| |PyPI Badge|
+
+.. |Maintainance Badge| image:: https://img.shields.io/badge/Status-Maintained-brightgreen
+   :alt: Maintainance Status
+.. |Test Badge| image:: https://img.shields.io/github/actions/workflow/status/edunext/eox-core/.github%2Fworkflows%2Ftests.yml?label=Test
+   :alt: GitHub Actions Workflow Test Status
+.. |PyPI Badge| image:: https://img.shields.io/pypi/v/eox-core?label=PyPI
+   :alt: PyPI - Version
+
+Eox-core is an `openedx plugin`_ for the `edx-platform`_, and part of the Edunext Open Extensions (aka EOX), that adds multiple API
+endpoints to extend its functionality and avoid changing the base code directly. These
+API endpoints include bulk creation of pre-activated users (for example, skip sending an activation email), enrollments, and pre-enrollment operations.
 
 Installation
 ============
 
-#. Add this plugin in your Tutor ``config.yml`` in the ``OPENEDX_EXTRA_PIP_REQUIREMENTS`` variable.
-#. Save your configuration with ``tutor config save``.
-#. Build your open edx image with ``tutor images build openedx``.
+#. Add this plugin in your Tutor ``config.yml`` with the ``OPENEDX_EXTRA_PIP_REQUIREMENTS`` setting.
+
+   .. code-block:: yaml
+      
+      OPENEDX_EXTRA_PIP_REQUIREMENTS:
+         - eox-core=={{version}} # basic installation
+         - eox-core{{requirements}}=={{version}} # requeriments e.g. [sentry,tpa]. Useful for integration with third-party applications.
+         
+#. Save the configuration with ``tutor config save``.
+#. Build an open edx image with ``tutor images build openedx``.
 #. Launch your platform with ``tutor local launch``.
 
 **Note:** To use all the features, you need to have `the tutor-forum plugin <https://github.com/overhangio/tutor-forum>`_ and `the eox-tenant plugin <https://github.com/eduNEXT/eox-tenant>`_.
 
 Features
 =========
-- Support redirections with middlewares.
+
+- Support redirections with middleware.
 - Add pipelines to be used with ``openedx-filters``.
```

Comment:
I don't think this is actually true. The pipelines here are for the social auth authentication pipeline, not for openedx filters.

Reply:
Could we update this line?